packages/@biomejs/js-api/scripts/update-beta-version.mjs

```diff
@@ -0,0 +1,23 @@
+import * as fs from "node:fs";
+import { resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+
+const BIOME_JS_API_ROOT = resolve(fileURLToPath(import.meta.url), "../..");
+const MANIFEST_PATH = resolve(BIOME_JS_API_ROOT, "package.json");
+
+const rootManifest = JSON.parse(fs.readFileSync(MANIFEST_PATH, "utf-8"));
+
+if (
+	typeof process.env.GITHUB_SHA !== "string" ||
+	process.env.GITHUB_SHA === ""
+) {
+	throw new Error("GITHUB_SHA environment variable is undefined");
+}
+
+const version = "0.8.0-beta";
```

Comment:
I think for now it works wonder. A later improvement might be to accept an input via workflow dispatch :)

Reply:
Yeah, I just mirrored the script for the CLI. Custom version input will be a great improvement.